readme.md

```diff
@@ -363,13 +363,15 @@ test(t => {
 });
 ```
 
-You can also use your own local Babel version:
+To use your own local Babel version, specify the version you want in your package.json, and ensure you're using npm@3 (or run `$ npm dedupe`), so the dependency will be deduped.
```

Comment:
No, I'm advocating dedupe so the user is able to set an explicit Babel 5 version that we will also use. Since we depend on a loose Babel 5 version, the user can set an exact version and it will be deduped to the user's choice.